code/datums/ai/basic_mobs/targetting_datums/basic_targetting_datum.dm

```diff
@@ -132,3 +136,12 @@
 
 /datum/targetting_datum/basic/attack_even_if_dead
 	stat_attack = DEAD
+
+/datum/targetting_datum/basic/attack_wounded_only
+	wounded_only = TRUE
+
+/// Intended mainly for "medic" mobs that have healing touch or some other support ability
+/datum/targetting_datum/basic/attack_wounded_only/same_faction
```

Comment:
Here's some examples of subtypes which should preferably just be blackboard key behaviour (including the stat_attack ones you didn't add)